README.md

```diff
@@ -465,16 +465,14 @@ Specific examples covered in the drop-down ▶ datasets above ⬆️:
 
 ## Citation
 
-If you use this for your work, please cite [this paper](https://arxiv.org/pdf/2103.14749.pdf):
+If you use this package for research, please cite [the paper](https://arxiv.org/abs/2103.14749):
 
 ```
-@misc{northcutt2021labelerrors,
+@inproceedings{northcutt2021labelerrors,
       title={Pervasive Label Errors in Test Sets Destabilize Machine Learning Benchmarks}, 
       author={Curtis G. Northcutt and Anish Athalye and Jonas Mueller},
       year={2021},
-      eprint={2103.14749},
-      archivePrefix={arXiv},
-      primaryClass={stat.ML}
+      booktitle={Proceedings of the Neural Information Processing Systems Track on Datasets and Benchmarks}
```

Comment:
Switch to "Proceedings of the 35th Conference on Neural Information Processing Systems (NeurIPS 2021) Track on Datasets and Benchmarks"?

Reply:
Where is that name from?  It has few results on google:
https://www.google.com/search?q=+%22Proceedings+of+the+35th+Conference+on+Neural+Information+Processing+Systems+%28NeurIPS+2021%29+Track+on+Datasets+and+Benchmarks%22&newwindow=1&sxsrf=APq-WBshcB1EwUCGIb1vxv9h1kAEa0i1_w%3A1645210978575&ei=Yu0PYsDAIt6-kPIProy48AU&ved=0ahUKEwjA-dS_-In2AhVeH0QIHS4GDl4Q4dUDCA4&uact=5&oq=+%22Proceedings+of+the+35th+Conference+on+Neural+Information+Processing+Systems+%28NeurIPS+2021%29+Track+on+Datasets+and+Benchmarks%22&gs_lcp=Cgdnd3Mtd2l6EAMyBwgAEEcQsAMyBwgAEEcQsAMyBwgAEEcQsAMyBwgAEEcQsAMyBwgAEEcQsAMyBwgAEEcQsAMyBwgAEEcQsANKBAhBGABKBAhGGABQlaYGWJWmBmDppwZoB3ABeACAAQCIAQCSAQCYAQCgAQKgAQHIAQfAAQE&sclient=gws-wiz

In contrast, the one I listed here has many hits:
https://neurips.cc/Conferences/2021/CallForDatasetsBenchmarks
https://ml.informatik.uni-freiburg.de/profile/vanrijn/
https://agents.inf.ed.ac.uk/publications/index.php?tags=NeurIPS

Reply:
Do you suggest the following?
"Proceedings of the 35th Conference on Neural Information Processing Systems Track on Datasets and Benchmarks"
Fine to go with that, it's just not the official name though (not sure how to add 35th into the official name):
https://neurips.cc/Conferences/2021/CallForDatasetsBenchmarks
https://datasets-benchmarks-proceedings.neurips.cc/paper/2021

Reply:
And did you mean `month` and `day` keys?  `year` is already in the bibtex right? 

Not sure what the month/day should be since the proceedings aren't out yet, any ideas?

Reply:
Oops yes, just month (can skip day, I think bib formats rarely use that). I'd just put the month of the conference.